oh-my-bash.bash

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
```

Comment:
@Kreyren The installation of `oh-my-bash` will create a template automatically and the filename inside it at this moment is `oh-my-bash.sh`, I suggest we create a merge script if someone upgrade from the previous versions their terminal won't mess up by missing sourced files.

Reply:
Sounds like bad practice to rely on script to make shabang. Forcing shabang everywhere should be a better way to handle this assuming situation where said file woudn't be executed correctly because it's missing said shabang. 

> I suggest we create a merge script 

i'm not confident in making this merge -> passing.

Recommend making it a part of the `install.bash`

Reply:
> The installation of `oh-my-bash` will create a template automatically and the filename inside it at this moment is `oh-my-bash.sh`, I suggest we create a merge script if someone upgrade from the previous versions their terminal won't mess up by missing sourced files.

Let's address it in #270. I feel we can just put a symbolic link from `oh-my-bash.sh` to `oh-my-bash.bash`. It is essentially impossible to correctly automatically update the generated file in the general case because users might edit the generated file.

> Sounds like bad practice to rely on script to make shabang. Forcing shabang everywhere should be a better way to handle this assuming situation where said file woudn't be executed correctly because it's missing said shabang.

Maybe I miss the point of the discussion, but I'm not sure whether we should add these shebangs to the script files which are sourced from other script files because they are not supposed to be executed by `exec(2)` syscalls.